nix/default.nix

```diff
@@ -120,10 +120,9 @@ in rec {
     chromedriver = pkgs.chromedriver;
 
     # Python development
-    pip3        = pkgs.python37Packages.pip;
-    python      = pkgs.python37Packages.python;
+    pip3        = pkgs.python38Packages.pip;
+    python      = pkgs.python38Packages.python;
     python3     = python;
-    python37 = python;
```

Comment:
This alias doesn't seem to be used anymore; I think there was a time where we had two different Python versions so this was useful. But no more.